src/components/ResultsMap.vue

```diff
@@ -29,11 +34,12 @@ export default {
     return {
 			markers: this.searchResults,
 			centerMap: {
-				lat: 40, lng: -105.0465427
+				lat: this.$store.state.geolocation.coords.latitude, 
```

Comment:
This is where the logic for centering the map based on the user's geolocation occures

Reply:
Great job with making this dynamic. I'm also glad putting geolocation in the store worked out for more than one component.

Reply:
Thank you for setting the center of the map dynamic based on the user's location info!